### PR TITLE
hawkular data mining first release

### PR DESCRIPTION
**Pull request by pavolloffay**
@Jiri-Kremser  could you please review?

### CODE REVIEW

File: src/main/jbake/content/blog/2016/04/21/datamining-first-release.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
+= Hawkular Data Mining 0.1.0.Final Released
+Pavol Loffay
+2016-04-21
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, hawkular, datamining
+
+I am happy to announce first release 0.1.0.Final of the Hawkular Data Mining project. It contains several time series
```
Comment (jkremser) [inline src/main/jbake/content/blog/2016/04/21/datamining-first-release.adoc]: the first

File: src/main/jbake/content/blog/2016/04/21/datamining-first-release.adoc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  - Time series lagging
+  - Time series differencing
+  - Automatic period identification
+
+== Integration into Hawkular
+
+The integration into Hawkular can be found in _datamining_ branch in the main Hawkular repository.
+Predictive charts are located in Explorer tab. There can be enable predictions for any number of steps in the future for
+ any metric being collected. Prediction engine automatically selects the best model for given time series. Currently it
+selects from simple, double and triple exponential smoothing models. In the following charts are
```
Comment (jkremser) [inline src/main/jbake/content/blog/2016/04/21/datamining-first-release.adoc]: In the following charts are showed predictions produced by these models.
In the following charts predictions produced by these models are showed .

File: src/main/jbake/content/blog/2016/04/21/datamining-first-release.adoc [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  - Autocorrelation function (ACF)
+  - Time series decomposition
+  - Time series lagging
+  - Time series differencing
+  - Automatic period identification
+
+== Integration into Hawkular
+
+The integration into Hawkular can be found in _datamining_ branch in the main Hawkular repository.
+Predictive charts are located in Explorer tab. There can be enable predictions for any number of steps in the future for
```
Comment (jkremser) [inline src/main/jbake/content/blog/2016/04/21/datamining-first-release.adoc]: There can be enable predictions for any number of steps..
Predictions can be enabled for any number of steps..

**Comment by jkremser** [comment]
:+1: graphs and results look great!

**Comment by pavolloffay** [comment]
Thanks, corrections are done.

**Pull request merged**